### PR TITLE
Convert project from CSS to SASS

### PR DESCRIPTION
**Pull request by Fosome**
Naming in buildout might be harder than naming in modeling...

Any feedback, tips, tricks, etc would be fosome!

### CODE REVIEW

**Comment by nhunzaker** [comment]
How does this deploy? Curious if we can ignore the compiled output

File: config.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Require any additional compass plugins here.
+
+# Set this to the root of your project when deployed:
+http_path = "/"
+css_dir = "css"
+sass_dir = "sass"
+images_dir = "img"
+javascripts_dir = "js"
+
+output_style = :expand
```
Comment (nhunzaker) [inline config.rb:11]: A couple of us set this to `:compressed` and use source maps to bridge the gap between the minified code and the original files. Check out sourcemaps in http://compass-style.org/help/documentation/configuration-reference/

Feel free to ping me if you have any questions.

**Comment by nhunzaker** [comment]
Seems güd. A couple of us use [autoprefixer](https://github.com/postcss/autoprefixer) instead of Compass's mixins to get browser compliance for things like Flexbox, but this can be its own :rabbit: :black_circle:. Still if you're curious, it's a cool tool that saves us a lot of work

**Comment by Fosome** [comment]
@nhunzaker This won't be deployed.  This is just static buildout for me to learn some new things.

File: config.rb (hunk 1 above)
Comment (Fosome) [inline config.rb:11]: Cool, I'll checkem.

**Comment by nhunzaker** [comment]
Cool. :+1:

**Pull request closed without merging**